examples/reshare/README.md

```diff
@@ -8,20 +8,37 @@ Reshare a threshold secret over an epoched log.
 
 _To run this example, you must first install [Rust](https://www.rust-lang.org/tools/install) and [`mprocs`](https://github.com/pvolok/mprocs)._
 
-## Usage
+## Usage (Trusted Setup)
 
 First, set up the network participants:
 
 ```sh
-# Default configuration (4 participants)
+# Default configuration (4 active participants, 2 inactive participants)
 cargo run --bin commonware-reshare setup
 
 # With configuration:
-cargo run --bin commonware-reshare setup [--num-peers <n>] [--num-bootstrappers <n>] [--datadir <path>] [--base-port <port>]
+cargo run --bin commonware-reshare setup [--num-peers <n>] [--num-bootstrappers <n>] [--base-port <port>]
 ```
 
 Then, run the `mprocs` command emitted by the setup procedure to start all participants simultaneously.
 
+## Usage (DKG Setup)
```

Comment:
I think we should just support DKG setup. There are already examples with trusted setup.